### DISCUSSION TITLE
Separate i18n prev/next_page for single posts and lists

### DISCUSSION THREAD

**Discussion by TymekDev** [adityatelange/hugo-PaperMod#235] [Ideas]
Right now there are i18n ids `prev_page` and `next_page` that are used both, at the bottom of a single page and in pagination. To me it sounds a bit unnatural to move to _Next/Previous Page_ when I am thinking about posts. If separated one can use _Next/Previous Page_ and _Next/Previous Post_.

What do you think about it?

**Comment by adityatelange** [comment]
Yeah I have initially thought about this, but getting strings translated is a very time consuming process. So I chose to use existing  available localization strings.
Also there can be cases where the website is not used for blog posts but for something else, `posts` may not be a generic term unlike `page`
